### ISSUE TITLE
Show thumnails

### ISSUE THREAD

**Issue by bjarnef** [biati-digital/glightbox#86]
**Please describe the feature you want to be implemented.**
It would a be great to enable the options to show thumbnails in the overlay, e.g. a combination of images and videoes (e.g. via a json-array). In this case it would be great to add a specific class in image/video thumnails to be able to style these, e.g. a play icon on video thumnails.

**Explain why the feature is useful**
It enable the option to see preview of prev/next slides in the gallery, which can be useful in a larger gallery.

**Have you seen it somewhere else?**
In Fancybox v3 it is possible in configuration to show thumbnails.

**Comment by biati-digital** [comment]
Not sure this will be added but I do like the way fancy box shows the thumbnails.

I'll leave this open to see if more people is interested in something like this.

**Comment by robertimes** [comment]
+ 1 for thumbs, also see https://github.com/sachinchoolur/lightGallery

**Comment by Loocos** [comment]
Yeah, It would be great if it can be added by default

**Comment by Loocos** [comment]
Alright! This sounds great, when u think it will be available (the plugin system) ? if we can help you to create some plugins

**Comment by biati-digital** [comment]
I've planned one more (minor) update "3.0.8" and then we'll be working on V4, this version will introduce the plugin system, I've no ETA yet, I'm finishing some projects so I've to focus on that first. I'll post a comment here once V4 is released.

**Comment by frankie-tech** [comment]
@biati-digital I'd love to help out with that plugin system, be it converting current features to plugins or with developing the system itself

**Comment by mohamedation** [comment]
@biati-digital any updates on this?

**Comment by biati-digital** [comment]
@mohamedation Yes, V4 is advancing rapidly, there's already a functional plugins system, hopefully someone implement this once v4 is out.

As soon as I've something more concrete I'll publish the branch

**Comment by projct1** [comment]
Agree, very wait this feature!
